unit_tests/k8s/test_remote_scylla_yaml.py

```diff
@@ -0,0 +1,106 @@
+import contextlib
+import yaml
+import pytest
+
+from sdcm.cluster_k8s import KubernetesCluster, SCYLLA_NAMESPACE
+
+
+class DummyCluster(KubernetesCluster):
+    POOL_LABEL_NAME = "dummy-pool"
+
+    def deploy(self):
+        pass
+
+    def create_kubectl_config(self):
+        pass
+
+    def create_token_update_thread(self):
+        return None
+
+    def deploy_node_pool(self, pool, wait_till_ready=True):
+        pass
+
+
+@pytest.fixture()
+def dummy_cluster():
+    params = {
+        "k8s_scylla_cluster_name": "test-cluster",
+        "cluster_backend": "k8s-local",
+    }
+    kl = DummyCluster(params=params, user_prefix="ut", region_name="region-1")
+    # Provide a simple in-memory config map and monkeypatch scylla_config_map
+    config_map_store = {}
+
+    @contextlib.contextmanager
+    def fake_scylla_config_map(namespace: str = SCYLLA_NAMESPACE):
+        assert namespace == SCYLLA_NAMESPACE
+        # Yield a real dict so context can mutate it
+        yield config_map_store
+        # No automatic patching to API; we're testing dict manipulation only
+
+    # Monkeypatch the method on instance
+    kl.scylla_config_map = fake_scylla_config_map  # type: ignore
```

Comment:
We do not need monkeypatch DummyCluster is our class, we can add it there on initialization

Reply:
move to the class